web/@linera/client/src/lib.rs

```diff
@@ -314,11 +321,21 @@ impl Client {
     ///
     /// If the owner is in the wrong format, or the chain client can't be instantiated.
     #[wasm_bindgen(js_name = addOwner)]
-    pub async fn add_owner(&self, owner: JsValue) -> JsResult<()> {
+    pub async fn add_owner(&self, owner: JsValue, options: JsValue) -> JsResult<()> {
+        #[derive(Default, serde::Deserialize)]
+        struct Options {
+            #[serde(default)]
+            weight: u64,
```

Comment:
Can you show the example how to construct this on the frontend side? As an example you can just pass instance with `weight: 0` here.